components/Home/styles.js

```diff
@@ -138,13 +138,15 @@ export const Boxlogo = styled('div')`
   }
 `;
 
-export const ImageBox = styled('div')`
+export const ImageBox = styled('div').attrs(props => ({
+  header: props.header ? 0 : '1rem',
+}))`
   margin-left: 1.5rem;
   margin-right: 1.5rem;
   padding: 0.5rem 0;
   @media (max-width: 37.5rem) {
     flex-direction: column;
-    margin-bottom: 1rem;
+    margin-bottom: ${props => props.header};
```

Comment:
Podemos hacer esa condición directamente en el template literal.
```suggestion
    margin-bottom: ${props => props.header ? 0 : '1rem'};
```

Reply:
Listo, ya lo cambié.